src/product/controllers/product-v1.controller.ts

```diff
@@ -29,4 +31,14 @@ export class ProductController {
       data: await this.productService.getProducts(),
     };
   }
+
+  @ApiOkResponse({type: ProductResponseDto})
+  @Delete()
+  async deleteProduct(@Query('productID') productID:string): Promise<DeleteProductResponse> {
```

Comment:
the variable name is not appropiated, "id" is better, if is minimal is better jajaja